cob_monitoring/src/ntp_monitor.py

```diff
@@ -136,14 +137,30 @@ def ntp_monitor_main(argv=sys.argv):
 
     if (len(args) != 2):
         parser.error("Invalid arguments. Must have HOSTNAME [args]. %s" % args)
-
+        print('Invalid arguments.', file=sys.stderr)
+        sys.exit(0)
 
     try:
         offset = int(options.offset_tol)
         self_offset = int(options.self_offset_tol)
         error_offset = int(options.error_offset_tol)
     except:
         parser.error("Offsets must be numbers")
+        print('Offsets must be numbers', file=sys.stderr)
+        sys.exit(0)
+
+    try:
+        rospy.init_node(NAME, anonymous=True)
+    except rospy.exceptions.ROSInitException:
+        print('NTP monitor is unable to initialize node. Master may not be running.', file=sys.stderr)
```

Comment:
is this really neccessary? I think we can assume that a roscore is already running as we do for all other nodes.

And in that case I'd like to use rospy.logerr/warn/info/debug for all print output.